doc/circuitikzmanual.tex

```diff
@@ -491,19 +491,11 @@ \subsection{A more complex tutorial: circuits, Romano style.}
 \end{circuitikz}
 \end{LTXexample}
 
-Another thing I like to modify with respect to the standard is the position of the arrows in transistors, which are normally in the middle the symbol. Using the following settings will move the arrows to the start or end of the corresponding pin.
+Another thing I like to modify with respect to the standard is the position of the arrows in transistors, which are normally in the middle the symbol. Using the following setting (see section~\ref{sec:styling-transistors}) will move the arrows to the start or end of the corresponding pin.
```

Comment:
doesn't the manual use `fancyref`?

Reply:
Dunno ;-) ...

Reply:
hmm it looks like it uses hyperref but not fancyref.  Nevermind then...one of these days I'll make a PR to replace all refs with Frefs

Reply:
Yep. Probably, while at it, we should change the document type from `article` to `book`... with chapters and appendixes --- it's quite big as an article already ;-)